File: Back/src/main/java/com/example/booking/project/models/entities/BookingModel.java
package com.example.booking.project.models.entities;

import jakarta.persistence.Column;

import java.time.LocalDate;

public class BookingModel {
    @Column
    private LocalDate end_date;
    @Column LocalDate start_date;
    @Column
    private int reserved_rooms;
    @Column
    private int no_guests;
    @Column
    private Long hotel_id;
    @Column
    private Long client_id;
    @Column
    private Long reservation_id;
}

File: Front/src/app/models/Booking.Model.ts
export class BookingModel {
    end_date: Date;
    start_date: Date;
    reserved_rooms: number;
    no_guests: number;
    hotel_id: number;
    client_id: number;
    reservation_id: number;


    constructor()
    {
        this.end_date = new Date;
        this.start_date = new Date;
        this.reserved_rooms = 0;
        this.no_guests = 0;
        this.hotel_id = 0;
        this.client_id = 0;
        this.reservation_id = 0;
    }
}